src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java

```diff
@@ -124,7 +126,10 @@ public void schedule(final Class< ? > taskClass) throws TaskServiceException
         LOGGER.info("Adding task {} to {}", taskName, this.getClass().getSimpleName());
 
         // Create JobDetail Object
-        JobDetail jobDetail = new JobDetail(taskName, DEFAULT_GROUP_NAME, QuartzTask.class);
+        JobDetail jobDetail = JobBuilder.newJob(QuartzTask.class) //
+            .withIdentity(taskName) //
+            .build();
```

Comment:
I was wrong. I thought that the jobs was scheduled in quartz DEFAULT_GROUP not in a custom one. I will fix it.